src/common/KokkosKernels_Utils.hpp

```diff
@@ -154,6 +155,104 @@ void get_suggested_vector_team_size(
 
 }
 
+#else
+template <typename idx, typename ExecutionSpace>
+void get_suggested_vector_size(
+    int &suggested_vector_size_,
+    idx nr, idx nnz){
+
+    suggested_vector_size_ =  1;
+
+#if defined( KOKKOS_ENABLE_SERIAL )
+  if (Kokkos::Impl::is_same< Kokkos::Serial , ExecutionSpace >::value){
```

Comment:
For later PRs (not this one :-) ), use `std::is_same` instead of `Kokkos::Impl::is_same`.

Reply:
I agree, it's existence here was artifact of copy/paste, I didn't look at all the content carefully.